src/client/Client.cc

```diff
@@ -8556,7 +8560,7 @@ int Client::readdir_r_cb(dir_result_t *d, add_dirent_cb_t cb, void *p,
       in = diri->get_first_parent()->dir->parent_inode;
 
     int r;
-    r = _getattr(in, caps, dirp->perms);
+    r = _getattr(in, mask, dirp->perms);
```

Comment:
Same here. Drop mask variable and just add CEPH_STAT_RSTAT to the argument inline.

Reply:
It's fixed，thanks!